### PR TITLE
Feature: Add featured on playlists and similar artists parsers

### PR DESCRIPTION
**Pull request by sunwize**
I noticed the artist page parser was missing 2 sections.
image

### CODE REVIEW

**Comment by sunwize** [comment]
@zS1L3NT

File: src/parsers/ArtistParser.ts [hunk 1]
```diff
@@ -60,4 +73,16 @@ export default class ArtistParser {
 			ArtistDetailed,
 		)
 	}
+
+	public static parseSimilarArtists(item: any): ArtistDetailed {
```
Comment (sunwize) [inline src/parsers/ArtistParser.ts:77]: Data structure commit back from the Youtube music API request was a bit different so I had to create a new parser.

File: src/parsers/PlaylistParser.ts [hunk 2]
```diff
@@ -43,4 +43,20 @@ export default class PlaylistParser {
 			PlaylistDetailed,
 		)
 	}
+
+	public static parseArtistFeaturedOn(item: any): PlaylistDetailed {
```
Comment (sunwize) [inline src/parsers/PlaylistParser.ts:47]: Same here

**Comment by zS1L3NT** [comment]
Thank you!!!

**Pull request merged**